newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/UserSurveyQuestion.js

```diff
@@ -0,0 +1,421 @@
+// @flow
+
+import * as React from 'react';
+import { Trans, t } from '@lingui/macro';
+import { I18n } from '@lingui/react';
+import { type I18n as I18nType } from '@lingui/core';
+import ButtonBase from '@material-ui/core/ButtonBase';
+import GridList from '@material-ui/core/GridList';
+import GridListTile from '@material-ui/core/GridListTile';
+import { darken, lighten, useTheme } from '@material-ui/core/styles';
+
+import { type AnswerData, type QuestionData } from './Questionnaire';
+import { Column, Line, Spacer } from '../../../../UI/Grid';
+import Text from '../../../../UI/Text';
+import TextField from '../../../../UI/TextField';
+import Paper from '../../../../UI/Paper';
+import InlineCheckbox from '../../../../UI/InlineCheckbox';
+import RaisedButton from '../../../../UI/RaisedButton';
+import {
+  useResponsiveWindowWidth,
+  type WidthType,
+} from '../../../../UI/Reponsive/ResponsiveWindowMeasurer';
+import { ColumnStackLayout } from '../../../../UI/Layout';
+import { isOnlyOneFreeAnswerPossible } from './UserSurvey';
+import { type MessageDescriptor } from '../../../../Utils/i18n/MessageDescriptor.flow';
+
+const getColumnsFromWidth = (width: WidthType) => {
+  switch (width) {
+    case 'small':
+      return 1;
+    case 'medium':
+      return 2;
+    case 'large':
+      return 3;
+    case 'xlarge':
+    default:
+      return 4;
+  }
+};
+
+export const TitleAndSubtitle = ({
+  i18n,
+  text,
+  multi,
+  answers,
+  textAlign,
+}: {
+  i18n: I18nType,
+  text: MessageDescriptor,
+  multi: ?boolean,
+  answers: AnswerData[],
+  textAlign: 'center' | 'left',
+}) => (
+  <>
```

Comment:
you could wrap this component in a i18n instead of passing it down from the top component

Reply:
I was fed up with adding the provider and I thought that it was not that bad to have it as a props, we use this in other places.
Do you think I should really do it?

Reply:
I'm not sure tbh, the advantage of using the provider here means it's easier to move around without bothering about passing the i18n too. 
But it will probably not be moved around, so I think it's fine!